### PR TITLE
Mention Couchbase SDK 2 Migration Kit

### PR DESCRIPTION
**Pull request by dnault**
The migration kit is a home for bits of code that might help users migrating from SDK 2 to SDK 3. It currently has a version of the experimental Query DSL from SDK 2, modified to work with SDK 3. It also has bridge classes for migrating subdoc requests.

### CODE REVIEW

**Comment by ingenthr** [comment]
Seems good by me.  Minor concern is there were concerns about how things are worded in the documentation to be cautious about implied support for those things (though you worded the README in the repo quite well).  I'll leave that concern up to @RichardSmedley.

**Review by osfameron — APPROVED** [review]
Agree with @ingenthr that the wording looks good.
As Richard is out, I'll approve to unblock if you want to merge now, and review later.

**Review by RichardSmedley — APPROVED** [review]
LGTM

**Pull request merged**
